cli/pcluster/config/validators.py

```diff
@@ -572,6 +572,24 @@ def url_validator(param_key, param_value, pcluster_config):
     return errors, warnings
 
 
+def s3_bucket_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if urlparse(param_value).scheme == "s3":
+        try:
+            bucket = param_value.split("/")[2]
+            boto3.client("s3").head_bucket(Bucket=bucket)
+        except ClientError:
+            warnings.append("The S3 bucket '{0}' does not exist or you do not have access to it.".format(param_value))
```

Comment:
It might be also useful at this point to add an assertion in the integration tests that verifies that no warnings are returned from the pcluster create command. This could be embedded in the ClustersFactory for now so that it gets applied to all tests, then we can add a flag to disable it in case some of the tests require a warning to be displayed.